Lavinia-api/Tests/ModelBuilderTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LaviniaApi.Models;
+using LaviniaApi.Utilities;
+using Xunit;
+
+namespace LaviniaApi.Tests
+{
+    public class ModelBuilderTests
```

Comment:
Tests are missing documentation and explanation for why they are sufficient. I don't see any failure-tests to ensure failures are handled properly, etc.